docs/docs/environment-variables.md

```diff
@@ -47,6 +47,10 @@ In addition to these Project Environment Variables defined in `.env.*` files, yo
 OS Env Vars. OS Env Vars which are prefixed with `GATSBY_` will become available in
 browser JavaScript.
 
+```shell:title=.env.*
```

Comment:
oh, does the name of this file actually include an asterisk?

Reply:
The asterisk means that all names are possible (it's a common way to write it like that), that's why it's already used in line 46.

Reply:
Yeah, @LekoArts I think this is reasonable.

@shannonbux for clarification, we typically use `.env.development` and `.env.production` since those credentials could (and often do) change between environments.